framework-core/src/main/java/org/daisy/pipeline/properties/Properties.java

```diff
@@ -0,0 +1,24 @@
+package org.daisy.pipeline.properties;
+
+import java.util.Map;
+
+public class Properties {
+
+	private final static java.util.Properties systemProperties = System.getProperties();
+	private final static Map<String,String> systemEnv = System.getenv();
```

Comment:
This works only if properties can't change dynamically. I think it's OK (?) but this should probably be documented in Javadoc?

Reply:
I didn't really care that much about dynamic changes when I wrote this. But if we can support it we probably should. So you say that the Properties object that `System.getProperties()` returns does not return a different value for `getProperty()` after a property was changed? In that case I should change it. And same question for `System.getenv()`?

Reply:
TBH I don't remember if `System.getProperties()` or `System.getenv()` are dynamic, so it needs to be checked first. If they aren't, then problem solved :)

Reply:
I checked, System.getProperties() is indeed dynamic. I'll add a comment. @rdeltour You meant that if they *are*, then problem solved, right?